meteor/client/ui/RundownList.tsx

```diff
@@ -297,6 +297,7 @@ export const RundownList = translateWithTracker(() => {
 											<span>{t('On Air Start Time')}</span>
 											<span>{t('Duration')}</span>
 											<span>{t('Last updated')}</span>
+											<span>{t('Shelf Layout')}</span>
 											<span>&nbsp;</span>
 										</header>
 										{this.renderRundownPlaylists(rundownPlaylists)}
```

Comment:
I would also like to see the support for this picker in "real" playlists (so ones that have 2+ rundowns in them). I don't see a reason for having a feature that has a "hole" in it like that, given that multi-rundown playlists are a first class citizen in Sofie.